pymeasure/adapters/ni_gpib_232ct.py

```diff
@@ -0,0 +1,346 @@
+            log.warning(f"ACHTUNG! {gpib_stat!a} \r\n")
+        # Error handling
+        if bool(gpib_stat & self.GPIBStatus.ERR) is True:
+            log.critical(f"Error {self.GPIBError(gpib_err)!a} {self.SERIALError(ser_err)!a}")
+
+    def _read(self, **kwargs):
+        """Read up to (excluding) `read_termination` or the whole read buffer.
+
+        :param kwargs: Keyword arguments for the connection itself.
+        :returns str: ASCII response of the instrument (excluding read_termination).
+        """
+        # log.debug("reading")
+        self.flush_read_buffer()
+        self.connection.write(f"rd #255 {self.address}")
+        time.sleep(0.050)
+        ret_val = self.connection.read()
+        if ret_val != "0":
+            ret_len = self.connection.read()
+            log.debug(f"length of read {ret_len}")
+            self._check_errors()
+        return ret_val
+
+    def _read_bytes(self, count, break_on_termchar, **kwargs):
+        """Read bytes from the instrument.
+
+        :param count: number of bytes to be read. A value of -1 indicates to
+            read from the whole read buffer (defined with the chunk_size).
+        :param bool break_on_termchar: Stop reading at a termination character.
+        :param kwargs: Keyword arguments for the connection itself.
+        :returns bytes: response of the instrument.
+        """
+        # log.debug("read bytes..")
+        if count == -1:
+            count = self.connection.chunk_size - 1
+        self.flush_read_buffer()
+        self.connection.write(f"rd #{count} {self.address}")
+        time.sleep(0.050)
+        ret_val = self.connection.read_bytes(count, break_on_termchar, **kwargs)
+        log.debug(f"read: {ret_val}")
+        time.sleep(0.050)
+        ret_len = self.connection.read()
+        log.debug(f"length of bytes read {ret_len}")
+        self._check_errors()
+        return ret_val
+
+    def _write(self, command, **kwargs):
+        """Write a string to the instrument appending `write_termination`.
+
+        :param str command: Command string to be sent to the instrument
+            (without termination).
+        :param kwargs: Keyword arguments for the connection itself.
+        """
+        self.flush_read_buffer()
+        self.connection.write(f"wrt {self.address} \n {command}", **kwargs)
+        time.sleep(0.050)
+        self._check_errors()
+        self.flush_read_buffer()
+
+    def _write_bytes(self, content, **kwargs):
+        """Write byte to the instrument appending `write_termination`.
+
+        :param str content: string to be sent to the instrument (without termination).
+        :param kwargs: Keyword arguments for the connection itself.
+        """
+        self.flush_read_buffer()
+        self.connection.write(f"wrt {self.address} \n {content}", **kwargs)
+        time.sleep(0.050)
+        self._check_errors()
+        self.flush_read_buffer()
+
+    def assert_trigger(self):
```

Comment:
```suggestion
    def trigger(self):
```
Wouldn't that be simpler?

Reply:
OK, then.